src/components/Table/components/TPagination/TPagination.style.ts

```diff
@@ -0,0 +1,60 @@
+import type { SerializedStyles, Theme } from '@emotion/react';
+import { css } from '@emotion/react';
+import { rem } from 'theme/utils';
+
+import type { TPaginationProps } from './TPagination';
+
+/** @TODO replace all styles with tokens */
+
+export const paginationContainer =
+  ({ isSticky }: Pick<TPaginationProps, 'isSticky'>) =>
+  (theme: Theme): SerializedStyles => {
+    return css`
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      background: ${theme.tokens.colors.get('backgroundColor.default')};
+      height: ${rem(44)};
+      width: 100%;
+      padding: 8px 16px;
```

Comment:
1) if this is fixed keep it in rem as pixel are not responsive
2) do we need tokens here?

Reply:
after merging master I will replace all these with tokens, I've put a TODO comment for now